packages/core/src/types-hoist/sdkinfo.ts

```diff
@@ -1,8 +1,14 @@
 import type { Package } from './package';
 
+/**
+ * See https://develop.sentry.dev/sdk/data-model/event-payloads/sdk/#attributes
+ */
 export interface SdkInfo {
   name?: string;
   version?: string;
   integrations?: string[];
   packages?: Package[];
+  settings?: {
+    infer_ip?: 'auto' | 'never' | 'legacy';
```

Comment:
we should probably omit 'legacy' here. No point to set this in SDKs, right?

```suggestion
    infer_ip?: 'auto' | 'never';
```

Reply:
Correct